<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="msgButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friendButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>好友</string>
    </property>
   </widget>
   <widget class="QPushButton" name="groupButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>群</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ChatManager">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>200</width>
      <height>600</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>需要自定义聊天显示控件1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>需要自定义聊天显示控件2</string>
     </property>
    </item>
   </widget>
   <widget class="QToolBox" name="FriendManager">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>200</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>548</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>好友</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>548</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QToolBox" name="GroupManager">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>200</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>548</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>群</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>548</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Page 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
